### PR TITLE
feat: Added utility functions to format duration and byte size for better d…

### PR DESCRIPTION
**Pull request by prasanthmj**
Created utility functions for formatting byte size to human readable format . Also added a function to format time duration for better display. 

Related to karina Pull request #762, issue #462

### CODE REVIEW

File: text/bytes.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package text
+
+/*
+Source from: https://github.com/cloudfoundry/bytefmt
+License: https://github.com/cloudfoundry/bytefmt/blob/master/LICENSE
+Apache License 2.0
```
Comment (moshloop) [inline text/bytes.go:6]: Can we make this a standard Apache File Header + Source

File: text/duration.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import (
+	"fmt"
+	"time"
+)
+
+//HumanizeDuration formats a time duration to human readable format
+//The duration in floored to the nearest unit. so 75 seconds becomes 1 minute
+//biggest unit is days (no weeks, months or years )
+func HumanizeDuration(duration time.Duration) string {
```
Comment (moshloop) [inline text/duration.go:11]: do we not need a license and attribution here ?

**Comment by moshloop** [comment]
Can you rebase and commit with a semantic version?

File: text/duration.go (hunk 2 above)
Comment (prasanthmj) [reply text/duration.go:11]: No. I wrote it.

File: text/bytes.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+// ByteSize returns a human-readable byte string of the form 10M, 12.5K, and so forth.  The following units are available:
+//	E: Exabyte
+//	P: Petabyte
+//	T: Terabyte
+//	G: Gigabyte
+//	M: Megabyte
+//	K: Kilobyte
+//	B: Byte
+// The unit that results in the smallest number greater than or equal to 1 is always chosen.
+func HumanByteSize(bytes uint64) string {
```
Comment (moshloop) [inline text/bytes.go:50]:
```suggestion
func HumanizeBytes(bytes uint64) string {
```

**Pull request merged**